cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -536,11 +536,19 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
                             block__border-top
                             block__flush-top
                             block__padded-top
-                            note-ctr">
-                    <div class="note">
-                        <p class="alert"><strong>Check your credit report.</strong></p>
-                        If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.
-                    </div> <!-- /.note -->
+                            credit-note-ctr">
+                    <div class="credit-note
+                                m-notification m-notification__warning
+                                m-notification__visible"
+                        role="alert">
+                        <span class="m-notification_icon cf-icon"></span>
+                        <div class="m-notification_content">
+                            <div class="h4 m-notification_message">Check your credit</div>
```

Comment:
Should this be `Check your credit report`?

Reply:
I"m not sure, it use to have the word `report` but it was removed in the mockup.